tests/utils/test_helpers.py

```diff
@@ -135,7 +134,7 @@ def test_temporary_directory_python_3_10_or_newer(mocker: MockerFixture) -> None
 
     mocker.patch.object(sys, "version_info", (3, 10))
     with temporary_directory() as tmp:
-        assert tmp
+        assert str(tmp)
```

Comment:
This `assert` has always been quite weak. Now it does not make any sense since the string representation of a `Path` will always be truthy as far as I know. We should probably do it similar to the test some lines below and set the return value of `mocked_temp_dir`. Then, we can check for the correct path.

Reply:
Fixed.